openapi.json

```diff
@@ -1797,7 +1871,7 @@
     "/credentials/basic": {
       "get": {
         "summary": "HTTP Basic authentication",
-        "description": "This request checks whether the credentials provided in the HTTP Basic header are valid. Returns the user ID and a bearer token for authorization in subsequent API calls. It is RECOMMENDED to implement this authentication method for non-public services only.",
+        "description": "This request checks whether valid credentials are provided through HTTP Basic Authentication (HTTP header `Authorization`, with type `Basic`).\n\nReturns an access token (without `basic//` prefix) to be used in the Bearer token for authorization in subsequent API calls. The Bearer Token sent to protected endpoints MUST be prefixed with `basic//`. The header would look as follows: `Authorization: Bearer basic//TOKEN` (replace `TOKEN` with the actual access token).\n\nIt is RECOMMENDED to implement this authentication method for non-public services only.\n\nAfter authentication, Clients SHOULD request further details about the user from `GET /me` so that they can make requests to the `/files/{user_id}` endpoints.",
```

Comment:
Maybe to avoid confusion, because you give a more concrete example about the `Authorization: Bearer` header than about the basic header:

> This request checks whether valid credentials are provided through HTTP Basic Authentication (HTTP header `Authorization`, with type `Basic`)

This request checks whether valid credentials are provided through HTTP Basic Authentication (HTTP header `Authorization: Basic ENCODEDCREDENTIALS`) and returns an access token.\n\nThe returned access token (which SHOULD NOT not include the `basic//` prefix) SHOULD be used in the Bearer token for authorization in subsequent API calls. ...

Reply:
I made another commit with even more clarifications. I hope it is crystal-clear now.